build.gradle

```diff
@@ -685,6 +685,7 @@ subprojects {
   }
 
   checkstyle {
+    configDirectory = rootProject.layout.projectDirectory.dir("checkstyle")
```

Comment:
is this rootDir.dir("checkstyle")? or `"File(rootDir, "checkstyle")"`?

My point is, could we make it relative to rootDir here?

Reply:
These all point to the same directory, though `rootProject.layout.projectDirectory.dir("checkstyle")` uses lazy properties, which are generally preferred.

Reply:
ok, that makes sense